packages/preview/miage-rapide-tp/0.1.0/template/main.typ

```diff
@@ -0,0 +1,84 @@
+#import "@local/miage-rapide-tp:0.1.0": *
```

Comment:
```suggestion
#import "@preview/miage-rapide-tp:0.1.0": *
```

Reply:
fixed in 5e20354fa3f87bbbfc94b9644421295b39e9b980